common/scripts/base.sh

```diff
@@ -99,8 +99,7 @@ function undeploy() {
 # $2 - service port
 function waitUntilServiceIsActive() {
   marathon_lb_host_ip=$(dcos marathon app show marathon-lb | $mesos_artifacts_home/common/scripts/get-marathon-lb-host.py)
```

Comment:
Shall we put a empty check validation here and also check the exit code of get-marathon-lb Python script by looking at $? variable?